routes/chars.js

```diff
@@ -4,17 +4,130 @@ var express = require('express'),
     importChar = npmarmory.importChar,
     Char = npmarmory.Char;
 
-// let's see if we can break node here.  ._. prelim logic
-// to import my monk
+// defaults
+var jadeRegionDefault = "US";
+var jadeNameDefault = "Caligraphy";
+var jadeRealmDefault = "Windrunner";
```

Comment:
You can clean this up this way:
var jadeRegionDefault = 'US',
   jadeNameDefault = 'Caligraphy',
   jadeRealmDefault = 'Windrunner';

Just a clean way to do variables. Also, JS best practice - always put your variables in the top!